src/shared/services/auth/auth.test.js

```diff
@@ -306,18 +331,15 @@ describe('The auth service', () => {
         access_token: '123AccessToken',
         token_type: 'token',
         expires_in: '36000',
-        state: '123StateToken',
+        state: 'random-string',
       };
-      savedStateToken = '123StateToken';
-      savedReturnPath = '/path/leading/back';
+      savedStateToken = 'random-string';
 
       initAuth();
-      expect(getReturnPath()).toBe(savedReturnPath);
     });
 
     it('returns an empty string when the callback was unsuccessful', () => {
       initAuth();
-      expect(getReturnPath()).toBe('');
```

Comment:
Not testing here anything anymore. Implicitly testing that it doesn't throw? You could make that explicit?

Reply:
Removed the whole function `getReturnPath()` from the codebase as it is not used any more. This part of 314a25a3491e8fd6eaf8fc12bd98cd3137c62364